--- bar-0.14/Makefile.in~	2010-07-28 07:07:33.000000000 +0200
+++ bar-0.14/Makefile.in	2010-07-30 23:13:42.130009654 +0200
@@ -172,6 +172,7 @@
 	bar/bar-keygen \
           --private-directory=. \
           --certs-directory=. \
+	  -d=./keys \
           --java-private-directory=. \
           $(if $(findstring $(KEYTOOL),certtool),--certtool) \
           $(if $(findstring $(KEYTOOL),openssl),--openssl) \
